compiler-core/src/ast.rs

```diff
@@ -313,6 +340,13 @@ pub enum Statement<T, Expr, ConstantRecordTag, PackageName> {
         typed_parameters: Vec<T>,
     },
 
+    /// Function from another language
```

Comment:
```suggestion
    /// Import a function defined outside of Gleam code.
    /// When compiling to Erlang the function could be implemented in Erlang
    /// or Elixir, when compiling to JavaScript it might be implemented in
    /// JavaScript or TypeScript.
```